components/_global-navigation.scss

```diff
@@ -75,6 +119,12 @@ $nav-width: 1024px;
 			width: 80px;
 		}
 
+		&.wds-has-multiple-subitems {
```

Comment:
Do we need a separate modifier for it? Maybe we could merge those styles into `content-bar`?